aunt-leahs-functions/shifts/index.js

```diff
@@ -112,7 +112,7 @@ module.exports = function (context, req) {
 
         request.on('doneProc', function (rowCount, more, rows) {
             context.res = {
-                body: { message: 'Successfully inserted 1 row into the database' }
+                body: { message: 'Successfully inserted 1 row into the database'}
```

Comment:
I feel like the client won't care about this message. I think we should return the actual message from the database instead. Or maybe instead just do  `success: true`?